### PR TITLE
User is now able to delete all the comments by clicking a button.

### PR DESCRIPTION
**Pull request by Cayle**
Implements a button that enables user to delete all the comments in the DataStore.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -34,6 +34,7 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+    static ArrayList<Long> commentIds = new ArrayList<Long>();
```
Comment (ljfranklin) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:37]: Static variable are almost always an anti-pattern. In Java this creates what amounts to a global variable. Google "global variables are bad" for some examples of why you should avoid them. Could we use the datastore instead of relying on globals here?

File: portfolio/src/main/java/com/google/sps/servlets/DeleteDataServlet.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.sps.data.Comments;
+// import com.google.sps.servlets.DataServlet;
```
Comment (ljfranklin) [inline portfolio/src/main/java/com/google/sps/servlets/DeleteDataServlet.java:26]: Delete dead code

File: portfolio/src/main/java/com/google/sps/servlets/DeleteDataServlet.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+        for (Long commentId : currentCommentIds) {
+            Key commentEntityKey = KeyFactory.createKey("Comments", commentId);
+            DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+            datastore.delete(commentEntityKey);
+        }
+        response.sendRedirect("/index.html");
+    }
+
```
Comment (ljfranklin) [inline portfolio/src/main/java/com/google/sps/servlets/DeleteDataServlet.java:56]: Run formatter to get rid of extra whitespace

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -103,3 +103,9 @@ async function getCommentCount(){
     commentContainer.innerHTML = commentHtml;
     
 }
+
+function deleteComments() {
+    fetch('/delete-data', {method: 'POST'});
+    window.location.reload();
+    // alert("yes");
```
Comment (ljfranklin) [inline portfolio/src/main/webapp/script.js:110]: Delete dead code

File: portfolio/src/main/webapp/script.js [hunk 5]
```diff
@@ -103,3 +103,9 @@ async function getCommentCount(){
     commentContainer.innerHTML = commentHtml;
     
 }
+
+function deleteComments() {
+    fetch('/delete-data', {method: 'POST'});
+    window.location.reload();
```
Comment (ljfranklin) [inline portfolio/src/main/webapp/script.js:109]: You can do this in a separate PR if you have time, but ideally you wouldn't need to reload the page to refresh the comments in the UI.

Comment (Cayle) [reply portfolio/src/main/webapp/script.js:109]: Oh, okay. I did that because the changes did not reflect on the UI immediately, but the comments were deleted in the DataStore.

Comment (Cayle) [reply portfolio/src/main/webapp/script.js:109]: I have implemented all the other issues raised in this PR.
